lib/src/components/NxToast/types.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import { HTMLAttributes, ReactNode, ReactElement } from 'react';
+import * as PropTypes from 'prop-types';
+
+export type Props = HTMLAttributes<HTMLDivElement>;
```

Comment:
What is this used by?

Reply:
> What is this used by?

Nothing now, so it has been removed here: [5277f2f](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/5277f2f440a29e543bf1cbde2fa2614027b96643). I also removed the generic propTypes below as I believe it's no longer necessary